models/Asset/Document.php

```diff
@@ -17,6 +17,7 @@
 
 use Pimcore\Cache;
 use Pimcore\Logger;
+use Pimcore\Messenger\AssetUpdateTasksMessage;
```

Comment:
Is this use necessary? 
At least my IDE says that it is never used.

Reply:
You are right, I removed it.